core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -58,10 +58,20 @@ class GroupMetadataManager(brokerId: Int,
                            interBrokerProtocolVersion: ApiVersion,
                            config: OffsetConfig,
                            val replicaManager: ReplicaManager,
-                           zkClient: KafkaZkClient,
+                           groupMetadataTopicPartitionCountFunc: () => Int,
                            time: Time,
                            metrics: Metrics) extends Logging with KafkaMetricsGroup {
 
+  def this(brokerId: Int,
```

Comment:
Do we need to have this glue code?  I only count 3 places where we create a GroupMetadataManager.